boards/arm/bl654_usb/bl654_usb.dts

```diff
@@ -8,8 +8,8 @@
 #include <nordic/nrf52840_qiaa.dtsi>
 
 / {
-	model = "Laird BL654 USB adapter";
-	compatible = "laird,bl654_usb";
+	model = "Laird Connectivity BL654 USB adapter";
+	compatible = "lairdconnect,bl654_usb";
```

Comment:
This change looks unrelated to the bug fix, please move it to a separate commit.